gazebo/Server.cc

```diff
@@ -194,6 +194,8 @@ bool Server::ParseArgs(int _argc, char **_argv)
      "Absolute path in which to store state data")
     ("record_period", po::value<double>()->default_value(-1),
      "Recording period (seconds).")
+    ("initial_sim_time", po::value<double>(),
+     "Initial simulation time (seconds).")
```

Comment:
I believe the order of these arguments determines the formatting of the help string, so I would move this out of the middle of the `record_*` arguments to be next to `iters`

Reply:
Fixed in https://github.com/gazebosim/gazebo-classic/pull/3294/commits/5cc4db31189963f21ce9e572427e7a03f42a6869.

Reply:
thanks; I pushed it one more line in https://github.com/gazebosim/gazebo-classic/pull/3294/commits/7f872b49088d85822df50626c60b2661cf0b4c8f